sql/alps_submitter_test.go

```diff
@@ -12,7 +12,7 @@ const (
 		WITH n_classes = 3 
 		COLUMN 
 			DENSE(c2, 5, comma), 
-			cross([BUCKET(NUMERIC(c1, 10), [1, 10]), NUMERIC(c4, 9), c5], 20) 
+			cross([BUCKET(NUMERIC(c1, 10), [1, 10]), c5], 20) 
```

Comment:
Should we change `cross` to `CROSS` to make the feature column naming consistent?

Reply:
The purpose of this is to check if resolver can handle lowercase correctly.